### PR TITLE
refactor(ui5-li-notification): wrap text by default

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
The title and description of `ui5-li-notification` now wrap by default.

BREAKING CHANGE: `wrapping-type` property default value has changed from `None` to `Normal`. Before:
```html
<ui5-li-notification title-text="Message 1">
	Description with details.
</ui5-li-notification>
<!-- would truncate the title and description if there is not enough space -->
```

Now:
```html
<ui5-li-notification title-text="Message 1">
	Description with details.
</ui5-li-notification>

<!-- would let the title and description wrap if there is not enough space -->
```

Related to https://github.com/SAP/ui5-webcomponents/issues/8461

### CODE REVIEW

File: packages/fiori/src/NotificationListItem.ts [hunk 1]
```diff
@@ -166,15 +166,15 @@ const ICON_PER_STATUS_DESIGN = {
...
 	/**
 	* Defines if the `titleText` and `description` should wrap,
-	* they truncate by default.
+	* they wrap by default.
 	*
-	* **Note:** by default the `titleText` and `description`,
-	* and a `ShowMore/Less` button would be displayed.
-	* @default "None"
+	* **Note:** when the property is set ot "None" and the texts truncate
+	* a `More/Less` button would be displayed.
```
Comment (BorisDafov) [inline packages/fiori/src/NotificationListItem.ts:172]: I suggest:
Defines if the `titleText` and `description` should wrap when there is not enough space. The possible values are:
•	"Normal" (default) - the texts should wrap.
•	"None" – the texts should truncate, and a `More/Less` button should be displayed.

**Review by BorisDafov — CHANGES_REQUESTED** [review]

**Review by BorisDafov — COMMENTED** [review]
I approve the PR from UA perspective

File: packages/fiori/test/pages/NotificationListGroupItem.html
Comment (GerganaKremenska) [inline packages/fiori/test/pages/NotificationListGroupItem.html:1]: Should we add wrapping-type="None", or just remove it so we can show case the new behavior? It looks a little wierd to change the default just so we can preset it with the old value?

Comment (LidiyaGeorgieva) [reply packages/fiori/test/pages/NotificationListGroupItem.html:1]: We will not change the default behavior for now.
I'm deleting this pull request.

**Pull request closed without merging**